### ISSUE TITLE
If the count of edge is beyond 268,435,455,how to create or load rr_graph?

### ISSUE THREAD

**Issue by szhou888** [verilog-to-routing/vtr-verilog-to-routing#1901]
According to capnp,the count of element need to be less than 268,435,455.How to deal the condition that edge number is beyond the limits?

#### Expected Behaviour

I want to load a large rr_graph with 303245222 edges.
#### Current Behaviour

Because the rr_graph's edge number is beyond the limits of capnp, and failed in loading the rr_graph.bin.

#### Possible Solution

How to expand the limit of edge number?

#### Your Environment

* VTR revision used: 8.1
* Operating System and version: ubuntu 18.04
* Compiler version:

**Comment by mithro** [comment]
@acomodi / @kgugala / @mkurc-ant -- What was our solution for this?

**Comment by acomodi** [comment]
I think the solution should be the one proposed in https://github.com/capnproto/capnproto/issues/1370#issuecomment-953067555. The issue we had were mainly related to the reading limit which, IIRC, can be pushed up to `2**63`, but this is should be inherent with a non-modifiable limit of elements in a list.

**Comment by szhou888** [comment]
> I think the solution should be the one proposed in [capnproto/capnproto#1370 (comment)](https://github.com/capnproto/capnproto/issues/1370#issuecomment-953067555). The issue we had were mainly related to the reading limit which, IIRC, can be pushed up to `2**63`, but this is should be inherent with a non-modifiable limit of elements in a list.

Yeah,following the suggestion to change the struct and the codes that access it!

**Comment by zhugesima** [comment]
How did you solve this problem, can you share it? Thanks very much!

**Comment by szhou888** [comment]
> How did you solve this problem, can you share it? Thanks very much!

You met the problem? Not good solution to solve this problem,the good solution just as [capnproto/capnproto/issues/1370](https://github.com/capnproto/capnproto/issues/1370#issuecomment-953067555) mentioned!
